lib/src/private/sub_extension/context/context_focus_extension.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+
+/// Extension on [BuildContext] to provide convenient methods for closing the keyboard.
+/// Example usage:
+/// context.unfocus;
+
+extension FocusExtension on BuildContext {
```

Comment:
the usage of 'kartal' is not acceptable. You should add it in the context_general extension, and then you need to call context.general.unFocus().